go.mod

```diff
@@ -6,6 +6,88 @@ require (
 	github.com/fxamacker/cbor/v2 v2.4.0
 	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/veraison/go-cose v1.0.0
+	golang.org/x/crypto v0.0.0-20220824171710-5757bc0c5503
```

Comment:
A tagged version of `golang.org/x/crypto` should be used. The current latest one is [v0.7.0](https://pkg.go.dev/golang.org/x/crypto?tab=versions)